pBoard:
- euler: [1, 1, 1]
  pos: [0, 0, 0]
- euler: [1, 1, 1]
  pos: [0, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [0, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [0, 3.25, 0]
- euler: [1, 1, 1]
  pos: [0, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [0, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [0, 6.5, 0]
- euler: [1, 1, 1]
  pos: [0, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [0, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [0, 9.75, 0]
- euler: [1, 1, 1]
  pos: [0, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [0, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [0, 13, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 0, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 3.25, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 6.5, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 9.75, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [1.0833333333333333, 13, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 0, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 3.25, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 6.5, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 9.75, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [2.1666666666666665, 13, 0]
- euler: [1, 1, 1]
  pos: [3.25, 0, 0]
- euler: [1, 1, 1]
  pos: [3.25, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [3.25, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [3.25, 3.25, 0]
- euler: [1, 1, 1]
  pos: [3.25, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [3.25, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [3.25, 6.5, 0]
- euler: [1, 1, 1]
  pos: [3.25, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [3.25, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [3.25, 9.75, 0]
- euler: [1, 1, 1]
  pos: [3.25, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [3.25, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [3.25, 13, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 0, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 3.25, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 6.5, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 9.75, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [4.333333333333333, 13, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 0, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 3.25, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 6.5, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 9.75, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [5.416666666666667, 13, 0]
- euler: [1, 1, 1]
  pos: [6.5, 0, 0]
- euler: [1, 1, 1]
  pos: [6.5, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [6.5, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [6.5, 3.25, 0]
- euler: [1, 1, 1]
  pos: [6.5, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [6.5, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [6.5, 6.5, 0]
- euler: [1, 1, 1]
  pos: [6.5, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [6.5, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [6.5, 9.75, 0]
- euler: [1, 1, 1]
  pos: [6.5, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [6.5, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [6.5, 13, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 0, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 3.25, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 6.5, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 9.75, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [7.583333333333333, 13, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 0, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 3.25, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 6.5, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 9.75, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [8.666666666666666, 13, 0]
- euler: [1, 1, 1]
  pos: [9.75, 0, 0]
- euler: [1, 1, 1]
  pos: [9.75, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [9.75, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [9.75, 3.25, 0]
- euler: [1, 1, 1]
  pos: [9.75, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [9.75, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [9.75, 6.5, 0]
- euler: [1, 1, 1]
  pos: [9.75, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [9.75, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [9.75, 9.75, 0]
- euler: [1, 1, 1]
  pos: [9.75, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [9.75, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [9.75, 13, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 0, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 3.25, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 6.5, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 9.75, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [10.833333333333334, 13, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 0, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 3.25, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 6.5, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 9.75, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [11.916666666666666, 13, 0]
- euler: [1, 1, 1]
  pos: [13, 0, 0]
- euler: [1, 1, 1]
  pos: [13, 1.0833333333333333, 0]
- euler: [1, 1, 1]
  pos: [13, 2.1666666666666665, 0]
- euler: [1, 1, 1]
  pos: [13, 3.25, 0]
- euler: [1, 1, 1]
  pos: [13, 4.333333333333333, 0]
- euler: [1, 1, 1]
  pos: [13, 5.416666666666667, 0]
- euler: [1, 1, 1]
  pos: [13, 6.5, 0]
- euler: [1, 1, 1]
  pos: [13, 7.583333333333333, 0]
- euler: [1, 1, 1]
  pos: [13, 8.666666666666666, 0]
- euler: [1, 1, 1]
  pos: [13, 9.75, 0]
- euler: [1, 1, 1]
  pos: [13, 10.833333333333334, 0]
- euler: [1, 1, 1]
  pos: [13, 11.916666666666666, 0]
- euler: [1, 1, 1]
  pos: [13, 13, 0]
pBoardCenter:
  euler: [0, 0, 0]
  pos: [0, 0, 0]
pButton:
  euler: [0, 0, 0]
  pos: [0, 0, 0]
pChess:
  euler: [0, 0, 0]
  pos: [0, 0, 0]
pHome:
  euler: [0, 0, 0]
  pos: [0, 12, 0]
